torchtune/models/qwen2/_model_builders.py

```diff
@@ -111,3 +157,109 @@ def lora_qwen2_7b(
         lora_dropout=lora_dropout,
         quantize_base=quantize_base,
     )
+
+
+def lora_qwen2_0_5b(
+        lora_attn_modules: List[LORA_ATTN_MODULES],
+        apply_lora_to_mlp: bool = False,
+        apply_lora_to_output: bool = False,
+        lora_rank: int = 8,
+        lora_alpha: float = 16,
+        lora_dropout: float = 0.05,
+        quantize_base: bool = False,
```

Comment:
nit: I think this is double-indented. Just kicked off CI so you can have linter signal but would also suggest running locally if you haven't already